platform/mds/mds-web/src/main/resources/webapp/js/directives.js

```diff
@@ -2268,9 +2268,7 @@
                         fieldPath = fieldPath.substring(fieldPath.indexOf('.') + 1);
                     }
 
-                    value = _.isBoolean(ngModel.$modelValue)
-                        ? !ngModel.$modelValue
-                        : ngModel.$modelValue;
+                    value = ngModel.$modelValue;
```

Comment:
@shanmukhm was there an issue only with boolean fields? The ticket does not mention that